packages/shared/test-suites/TestRun.ts

```diff
@@ -53,6 +56,8 @@ export type TestExecution = {
   errors: string[] | null;
   createdAt: string;
   result: string;
+  commitTitle: string;
+  recordings: (Recording | null)[];
```

Comment:
```suggestion
  recordings: Recording[];
```

`null` Recordings would be filtered now.

Reply:
good catch — thanks!